Delectio/translator-backend
# Setup basic node project with express

#### What does this PR do?
Setup basic node project with express

#### Description of Task to be completed?
- Setup babel
- Setup express app
- Add base route

#### How should this be manually tested?
- Clone the repository
- On your terminal, start the server by running `npm run start` or use `npm run start:dev` to run in development mode
- The server should start on port 8080 or 3000
- Open localhost:8080 on your browser
- You should get a json response to indicate that you successfully connected to the api

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?(if applicable)
NA

#### Screenshots
NA
